README.md

```diff
@@ -1,2 +1,184 @@
-# budget-me-App
-Budget Me App is a fully functioning website that allows a user to fully manage his/her expenses by keeping track of all payments and their related categories. For each category the user can see how much is already spent. A user can add categories and payments related to one or more categories. 
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Deployment](#setup)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Budgt Me App] <a name="about-project"></a>
+
+**[Budgt Me App]** Budget Me App is a fully functioning website that allows a user to fully manage his/her expenses by keeping track of all payments and their related categories. For each category the user can see how much is already spent. A user can add categories and payments related to one or more categories.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://www.ruby-lang.org/en/">Rails</a></li>
+    <li><a href="https://www.ruby-lang.org/en/">Postgresql</a></li>
+  </ul>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Creating tables by migration files]**
+- **[Creating Database named Budget]**
+- **[processing data in models]**
+- **[Validations and Model specs]**
+- **[Setup controllers]**
+- **[Controllers specs]**
+- **[Categories and payment views]**
+- **[Create new category]**
+- **[Create new payment]**
+- **[Add-devise]**
+- **[Add-authorization rules]**
+- **[Style the views (Pages)]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Deployment <a name="getting-started"></a>
+- Presentation: [Presentation](https://drive.google.com/file/d/1Y0sH7urTL8KB-E9qrLPqthG1H7g1D2u1/view?usp=sharing)
+- Deployment: [Deployment](https://budget-app-gwxd.onrender.com)
```

Comment:
- [ ] Great job implementing some required changes from the previous reviewer. However, there's one major required change you're yet to work on. Kindly work it so that you can move to the next milestone in your journey. According to the project requirement, you're to highlight a piece of code you were proud of implementing during your project development. That is not yet included in your video when I watched it. Kindly re-make a new video and ensure a section where you show a piece of code you are very proud of implementing while building the app. It might be a challenging but which gave you an opportunnity to learn one or two things.